### ISSUE TITLE
Update Button Styles

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#400]
We don't have many instances of buttons used on the portal, but where we are using them, they:

- use inconsistent hover styles
- lack an active state
- break color contrast guidelines for 508 accessibility.

**Comment by anilnatha** [comment]
I've updated color styles for the buttons used in carousel slides, the arrow navigation, and the pill button. The hover state color for the buttons in the slides was failing color contrast guidelines, but once that was fixed, the other elements in the carousel needed to be updated for consistency.

The chance has been applied to GCC Ops. I'm unable to access UAT due to login issues, but once that is up, I'll apply the changes there as well.

**Comment by anilnatha** [comment]
Button styles for carousels are synchronized for all three environments.
